apps/server/src/controllers/groupControllers/createGroup.ts

```diff
@@ -126,6 +126,11 @@ export const createGroup = async (req: RequestExt, res: Response) => {
           { transaction }
         )
 
+    await GroupWallet.create({
+      id: v4(),
+      groupId: groupId,
+      balance: 0,
+    })
```

Comment:
There's already an instance of GroupWallet creation just right above this code snippet. So why create the GroupWallet again??